README.md

```diff
@@ -1,4 +1,4 @@
-# curriculum-cr-training-contact-form
+# contact-form
 
 This repo host an initial code for the tasks performed during the Code Reviewers training.
 
```

Comment:
Please add a descriptive Readme file, You can use [this template](https://github.com/microverseinc/readme-template)  as It is a general requirement and ignoring it would invalidate the review